### ISSUE TITLE
Quick create form tabs on project detail view subpanels do not work

### ISSUE THREAD

**Issue by SinergiaCRM** [salesagility/SuiteCRM#8780]
#### Issue
If a quick creation form contains multiple tabs, is not possible switch between them, if it is in **any** subpanel of the **Project** module **detail view**.
The error does not occur when opening the same quickcreate form in another module.
Apparently there is a javascript error that prevents switching between tabs. The browser console shows the following information:

```
error adding script
sugar_grp1.js?v=_4iGDDL7V2JzxREFEHma0A:436 SyntaxError: Identifier 'selectTabOnError' has already been declared
    at eval (eval at <anonymous> (sugar_grp1.js?v=_4iGDDL7V2JzxREFEHma0A:428), <anonymous>:1:1)
    at eval (<anonymous>)
    at sugar_grp1.js?v=_4iGDDL7V2JzxREFEHma0A:428
    at Object.globalEval (sugar_grp1.js?v=_4iGDDL7V2JzxREFEHma0A:428)
    at Object.evalScript (sugar_grp1.js?v=_4iGDDL7V2JzxREFEHma0A:435)
    at Object.success (SubPanelTiles.js?v=_4iGDDL7V2JzxREFEHma0A:100)
    at Object.handleTransactionResponse (sugar_grp1_yui.js?v=_4iGDDL7V2JzxREFEHma0A:109)
    at sugar_grp1_yui.js?v=_4iGDDL7V2JzxREFEHma0A:109
```

This happens in the same way if the quickcreate form is from a SuiteCRM own module or a custom module
#### Expected Behavior
It should be possible to switch between tabs.
#### Actual Behavior
It is not possible to switch between tabs in a subpanel of the Projects module
#### Steps to Reproduce
1. Under studio, modify the quickcreate layout of the Tasks module so that the fields are distributed in two or more tabs (not panels).
2. Open a Project record in detail view.
3. In the Activities subpanel, create a new task by clicking on Create Tasks.
4. Try to switch between the tabs of the quickcreate form. It does not work.

If the test is performed with a subpanel of another stock or custom module, the error will still occur
#### Context
Some custom modules contain many fields and it is necessary to distribute the information in tabs.
#### Your Environment
SuiteCRM Version used: 7.11.11 (and SugarCRM 6.5)
Browser name and version: Chrome 83.0.4103 and Firefox 77.0.1 (64-bit) 
Environment name and version: PHP 7.3, MySQL
Operating System and version: Debian 9

**Comment by hassanbinaziz** [comment]
When AJAX UI Enabled tabs don't work in subpanels anywhere.

**Comment by SinergiaCRM** [comment]
Duplicated

**Comment by Esteban200589** [comment]
For me
I am getting this error in the AOS_Products module
when creating a new record in one of the sub-panels (currently there are two) with quick create view
I've created a custom module and made a relationship between aos_products and the custom module, but it doesn't just happen on the quickcreate of that module.
It also happens in the quick create of the other subpanels.
I don't have tabs in quick create and there aren't many fields.

In the metadata/quickcreatedefs.php I included a js:
       'includes' =>
       array(
         0 =>
         array(
           'file' => 'custom/modules/infor_components/quickcreate.js',
         ),
       ),

but remove this to see if it messed with the problem and no consequence

I need help!
